### PR TITLE
Add tests for IBMCloud

### PR DESCRIPTION
**Pull request by liangxia**
/cc @MayXuQQ @jhou1 @JianLi-RH @dis016 @pruan-rht

### CODE REVIEW

File: features/admin/project.feature [hunk 1]
```diff
@@ -4,8 +4,8 @@ Feature: project permissions
   # @case_id OCP-11717
   @admin
   @4.12 @4.11 @4.10 @4.9 @4.8 @4.7 @4.6
-  @vsphere-ipi @openstack-ipi @nutanix-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi @alicloud-ipi
-  @vsphere-upi @openstack-upi @nutanix-upi @gcp-upi @baremetal-upi @azure-upi @aws-upi @alicloud-upi
+  @vsphere-ipi @openstack-ipi @nutanix-ipi @ibmcloud-ipi @gcp-ipi @baremetal-ipi @azure-ipi @aws-ipi @alicloud-ipi
+  @vsphere-upi @openstack-upi @nutanix-upi @ibmcloud-upi @gcp-upi @baremetal-upi @azure-upi @aws-upi @alicloud-upi
```
Comment (MayXuQQ) [inline features/admin/project.feature:8]: has not ibmcloud upi

Comment (liangxia) [reply features/admin/project.feature:8]: Pre-add the tag, but not used at the moment.

**Pull request merged**